### PR TITLE
Bratseth/constants cleanup 2 take 2

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java [hunk 1]
```diff
@@ -18,8 +18,11 @@
 public class ModelsEvaluatorTest {
 
     @Test
-    public void testModelsEvaluatorTester() {
+    public void testModelsEvaluator() {
+        // Assumption fails but test passes on Intel macs
+        // Assumption fails and test fails on ARM64
```
Comment (lesters) [inline config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java:23]: Confusing comment, so this test always fails?

**Review by lesters — APPROVED** [review]
LGTM, but the comment in `testModelsEvaluator` is a bit confusing to me

File: config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java:23]: It says "test passes" so I don't know how you summarize it as "always fails".
Anyway, it leaves out linux intel, where (I assume) both the assumption and test succeeds.

Comment (lesters) [reply config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java:23]: I understand "assumption fails" as that `assumeTrue` fails, and it was confusing how the test then can pass?

Comment (bratseth) [reply config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java:23]: Yes, that was surprising to me as well. I guess OnnxEvaluator.isRuntimeAvailable() is wrong.

**Pull request merged**